observable_operator_test.go

```diff
@@ -607,6 +607,95 @@ func Test_Observable_IgnoreElements_Parallel_Error(t *testing.T) {
 	Assert(context.Background(), t, obs, IsEmpty(), HasError(errFoo))
 }
 
+func Test_Observable_Join(t *testing.T) {
```

Comment:
The position should be after the tests on `GroupBy`

Reply:
Moved below